Cubical/Data/Sigma/Properties.agda

```diff
@@ -115,7 +120,9 @@ discreteΣ {B = B} Adis Bdis (a0 , b0) (a1 , b1) = discreteΣ' (Adis a0 a1)
         discreteΣ'' : (b1 : B a0) → Dec ((a0 , b0) ≡ (a0 , b1))
         discreteΣ'' b1 with Bdis a0 b0 b1
         ... | (yes q) = yes (transport ΣPath≡PathΣ (refl , q))
-        ... | (no ¬q) = no (λ r → ¬q (subst (λ X → PathP (λ i → B (X i)) b0 b1) (Discrete→isSet Adis a0 a0 (cong fst r) refl) (cong snd r)))
+        ... | (no ¬q) = no (λ r → ¬q (subst (λ X → PathP (λ i → B (X i)) b0 b1)
+                                            (Discrete→isSet Adis a0 a0 (cong fst r) refl)
+                                            (cong snd r)))
```

Comment:
I started converting long lines to shorter ones, but it's too many files for one commit.

Reply:
Yeah, that should go in a separate PR that does only that